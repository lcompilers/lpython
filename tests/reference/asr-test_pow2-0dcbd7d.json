{
    "basename": "asr-test_pow2-0dcbd7d",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pow2.py",
    "infile_hash": "2e0d9090938bea6ffb7fe4c1cf11578ea03b99b9124bd9685a65aa6c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_pow2-0dcbd7d.stdout",
    "stdout_hash": "b57ae6b272fa90b8a017351338fb12283263567c0d1df62837eab86f",
    "stderr": "asr-test_pow2-0dcbd7d.stderr",
    "stderr_hash": "dfeb54e69d6104e760eed623a5df4d0d4617b7571278815ab804c297",
    "returncode": 0
}