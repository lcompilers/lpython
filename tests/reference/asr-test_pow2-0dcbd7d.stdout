(TranslationUnit (SymbolTable 1 {lpython_builtin: (IntrinsicModule lpython_builtin), main: (Subroutine (SymbolTable 2 {pow: (ExternalSymbol 2 pow 4 pow lpython_builtin [] pow Private)}) main [] [(Print () [(FunctionCall 1 pow@__lpython_overloaded_0__pow 2 pow [((IntegerConstant 2 (Integer 4 []))) ((IntegerConstant 2 (Integer 4 [])))] (Integer 4 []) (IntegerConstant 4 (Integer 4 [])) ())] () ()) (Print () [(IntegerBinOp (IntegerConstant 2 (Integer 4 [])) Pow (IntegerConstant 2 (Integer 4 [])) (Integer 4 []) (IntegerConstant 4 (Integer 4 [])))] () ())] Source Public Implementation () .false. .false.), main_program: (Program (SymbolTable 79 {}) main_program [] []), pow@__lpython_overloaded_0__pow: (ExternalSymbol 1 pow@__lpython_overloaded_0__pow 4 __lpython_overloaded_0__pow lpython_builtin [] __lpython_overloaded_0__pow Public)}) [])
